nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -266,6 +266,9 @@ SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "{{ secret_key }}")
 # CELERY_TASK_SOFT_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_SOFT_TIME_LIMIT", str(5 * 60)))
 # CELERY_TASK_TIME_LIMIT = int(os.getenv("NAUTOBOT_CELERY_TASK_TIME_LIMIT", str(10 * 60)))
 
+# Use threads Celery pool - required for git repository job support
+# CELERY_WORKER_POOL = "threads"
+
```

Comment:
If this isn't something we are going to recommend/support changing, it can probably be omitted from this template.